### DISCUSSION TITLE
CSS selector support : ~ and :target

### DISCUSSION THREAD

**Discussion by kitepad** [c-smile/sciter-js-sdk#124] [Ideas]
Recently, I'm trying to work my UI prototype using pure css tab control. I find CSS selector "~" is used in many solutions, but sciter.js don't support this selector.  Can Sciter.js provide ~ selector support? 

It's great help for reusing existing CSS code and  can help Sciter.JS ecosystem growth faster.

And CSS 3 :target selector is another to simply UI layout. Below is sample code running in browser.
<html>
<head>
<style>
.tab div {
  display: none;
}

.tab div:target {
  display: block;
}
</style>
</head>
<body>

<div class="tab">

<a href="#link1">Link 1</a>   
<a href="#link2">Link 2</a>
<a href="#link3">Link 3</a>

<div id="link1">
  <h3>Content to Link 1</h3>
  <p>Hello World!</p>
</div>

<div id="link2">
  <h3>Content to Link 2</h3>
  <h4>Great success!</h4>
</div>

<div id="link3">
  <h3>Content to Link 3</h3>
  <p>Yeah!</p>
</div>

</div>

</body>
</html>

**Comment by c-smile** [comment]
Sciter already supports '~' selector.
